scripts/griddle.jsx

```diff
@@ -153,6 +154,54 @@ var Griddle = React.createClass({
            return false;
        });
     },
+
+    filterByColumnFilters(columnFilters) {
+      var filteredResults = Object.keys(columnFilters).reduce(function(previous, current) {
```

Comment:
Here, we reduce the data by the columnFilters. The result of filtering the data by one column is passed into the next iteration of reduce (as previous).